chutney/ui/src/app/modules/campaign/components/campaign-scheduling/campaign-scheduling.component.html

```diff
@@ -169,6 +174,9 @@ <h4>{{ 'campaigns.scheduling.title' | translate }}</h4>
                         @if (campaignExecutionRequest.datasetId) {
                         ({{campaignExecutionRequest.datasetId}})
                         }
+                        @if (campaignExecutionRequest.jiraId) {
```

Comment:
<img width="1083" height="91" alt="image" src="https://github.com/user-attachments/assets/c2fff317-00bb-42a2-8c18-5688aaf3457c" />
I prefer for dataset and jira to have in front of ids a icon indicating the type of id.
For dataset, use fa-table and jira, use fa-jira.